### ISSUE TITLE
from ... import * tried to import flog

### ISSUE THREAD

**Issue by bobbyy98** [ucbrise/flor#76]
Some wildcard imports raise AttributeErrors on flor exec, e.g.
```
File "/Users/bobby/miniconda3/envs/flor/lib/python3.7/site-packages/scipy/sparse/linalg/__init__.py", line 3, in <module>
    from .isolve import *
AttributeError: module 'scipy.sparse.linalg.isolve' has no attribute 'flog'
```

Bug is caused by some transformed library code having the following structure:
```
from flor import Flog
if Flog.flagged():
    flog = Flog()
...
__all__ = [s for s in dir() if not s.startswith('_')]
```
with `dir()` returning all names in the current scope and having `flog` included in `__all__`.

**Comment by bobbyy98** [comment]
Fix in ab8388b34078a344f12c35914a252ea885e455bb

**Comment by bobbyy98** [comment]
The `__all__` patch was not fully general. A more general patch with static analysis in
https://github.com/ucbrise/flor/blob/5950903b0e01ad2efcb279bd809ac879cbb582f3/flor/complete_capture/transformer/transformer.py#L148-L155
